platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/extensions/DefaultMetricConfig.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.metrics.extensions;
+
+import static com.swirlds.common.utility.CommonUtils.throwArgBlank;
+
+import com.swirlds.common.metrics.Metric;
+
+/**
+ * A default configuration for a metrics with values that all metrics need
+ */
+public class DefaultMetricConfig {
```

Comment:
There is actually something very similar coming up with the custom metrics API, though it will be slightly different. Therefore I would like to avoid adding this class at this point.

Reply:
This class is just a wrapper around three strings, I don't think it is that crucial right now. If you want to keep it, you can make it an inner class of `BusyTime` as it is only used there as far as I can see.

Reply:
removed